planning/behavior_velocity_planner/src/scene_module/crosswalk/scene_crosswalk.cpp

```diff
@@ -735,6 +741,53 @@ CollisionPointState CrosswalkModule::getCollisionPointState(
   return CollisionPointState::IGNORE;
 }
 
+void CrosswalkModule::applySafetySlowDownSpeed(PathWithLaneId & output)
+{
+  const auto & ego_pos = planner_data_->current_pose.pose.position;
+  const auto ego_path = output;
+
+  // Safety slow down speed in [m/s]
+  const auto & safety_slow_down_speed =
+    static_cast<float>(crosswalk_.attribute("safety_slow_down_speed").asDouble().get());
+
+  if (!passed_safety_slow_point_) {
+    // Safety slow down distance
+    double safety_slow_down_distance = 2.0;  // [m]
+    if (crosswalk_.hasAttribute("safety_slow_down_distance")) {
+      safety_slow_down_distance =
+        crosswalk_.attribute("safety_slow_down_distance").asDouble().get();
+    }
```

Comment:
You should be able to simplify this as the following ([reference]( https://github.com/fzi-forschungszentrum-informatik/Lanelet2/blob/700a96290c9782654e3cf4c287ab1ddec2b7cfbe/lanelet2_examples/src/01_dealing_with_lanelet_primitives/main.cpp#L81)):
```suggestion
    double safety_slow_down_distance = 
        crosswalk_.attributeOr("safety_slow_down_distance", 2.0);
```

Reply:
Thank you for the simplified fix proposal. Will fix it.